inc/apirest.class.php

```diff
@@ -485,7 +493,7 @@ public function parseIncomingParams($is_inline_doc = false) {
     *
     * @return void
     */
-   public function returnResponse($response, $httpcode=200, $additionalheaders=array()) {
+   protected function returnResponse($response, $httpcode=200, $additionalheaders=array()) {
```

Comment:
this access mode does not matches the overriden method in the class API, then fixing.